test/unit/modules/blocks/processTablesData.json

```diff
@@ -0,0 +1,364 @@
+[
```

Comment:
Within a .json file is not possible to add comments like // or /**/.
I can rename the file to js and adjust to allow comments if you want.